snippets/settings.py

```diff
@@ -44,6 +43,7 @@
     'smuggler',
     'django_filters',
     'django_ace',
+    'product_details',
```

Comment:
Yeah, I guess you can just set `CACHE_URL` to a `redis://` URL and it'll work. Nice :)